website/blog/2018-05-23-1.13.0.md

```diff
@@ -1,17 +1,36 @@
 ---
 author: Lucas Duailibe (@duailibe)
 authorURL: https://twitter.com/duailibe
-title: "Prettier 1.13: TODO PUT A TITLE HERE"
+title: "Prettier 1.13: Conquering the web!"
 ---
 
-This releases features a lot of formatting fixes and changes.
+This releases adds a new  support for several new features and first-class support for working on the browser.
 
 <!--truncate-->
 
 ## Highlights
 
 ### API/CLI
 
+#### Prettier works on the browser!
+
+This has been long wanted by the team and our users and we're finally announcing Prettier 1.13 has first-class support for running in the browser. In the past it required several hacks like (defining globals like `os`, `path` and `require`). Just load the core bundle and the parsers you want to use.
+
+<!-- prettier-ignore -->
+```html
+<script src="https://unpkg.me/prettier@1.13.0/standalone.js"></script>
+<script src="https://unpkg.me/prettier@1.13.0/parser-babylon.js"></script>
+<script src="https://unpkg.me/prettier@1.13.0/parser-graphql.js"></script>
+<script type="text/javascript">
+  $("#format").onClick(function() {
```

Comment:
Been I while since I jQueried, but isn't it `$(sel).click(fn)`?

Reply:
It is. But that's appears to be deprecated: https://github.com/jquery/api.jquery.com/issues/972
I'd go with `$(sel).on("click", fn)`.

Reply:
Or `document.querySelector(sel).addEventListener(fn)`  😆

Reply:
Yeah I tried to be smart here and my React leaked 😂